### ISSUE TITLE
Partition key is absent on export of cookies from firefox

### ISSUE THREAD

**Issue by igvk** [Rob--W/cookie-manager#61]
When there are partitioned cookies, and their export to file (or text) is selected, they are exported with absent partition key.
It either null, or empty (consists of {} braces only, without value).
Example:
```
 {
  "name": "cookie",
  "value": "",
  "domain": "domain.com",
  "hostOnly": true,
  "path": "/",
  "secure": true,
  "httpOnly": false,
  "session": false,
  "storeId": "firefox-container",
  "sameSite": "no_restriction",
  "firstPartyDomain": "",
  "partitionKey": {}
 }
```

**Comment by Rob--W** [comment]
Thanks for your report! I can reproduce it, e.g. by visiting https://example.com and running `new Image().src = "https://stackoverflow.com/"` from the console.

While the UI displays the partitioned cookies, the exported cookie lacks the expected `topLevelSite` key.

This happens because I'm explicitly listing the set of permitted keys as part of JSON serialization, and apparently nested objects are also subjected to the list of permitted keys:

https://github.com/Rob--W/cookie-manager/blob/c584e26921d90d81c082225d34183ee9b2582cdc/cookie-manager.js#L948

I'll work on a fix soon.

**Comment by Rob--W** [comment]
Fixed in version 1.8.

**Comment by igvk** [comment]
That's great you fixed it!
